homeassistant/components/mqtt_statestream/__init__.py

```diff
@@ -41,15 +41,6 @@
 
 async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     """Set up the MQTT state feed."""
-    # Make sure MQTT is available and the entry is loaded
-    if not hass.config_entries.async_entries(
-        mqtt.DOMAIN
-    ) or not await hass.config_entries.async_wait_component(
-        hass.config_entries.async_entries(mqtt.DOMAIN)[0]
-    ):
-        _LOGGER.error("MQTT integration is not available")
-        return False
-
     conf: ConfigType = config[DOMAIN]
```

Comment:
We should do this in another PR once the core changes merge

Reply:
I do not understand, the other PR was merged, this one was rebased.

Reply:
But I can split these changes to another PR.